public/app.js

```diff
@@ -0,0 +1,380 @@
+      },
+      body: 'grant_type=client_credentials'
+    }).catch(err => alert(err.message));
+
+    const data = await result.json();
+
+    if (data.error) {
+       alert("We cant find your token :(");
+      return null;
+    } else {
+      return data.access_token;
+      }
+
+  }
+  /**
+   * @description Функция получения жанров
+   * @param {string} token - токен Spotify
+   */
+  const _getGenres = async (token) => {
+    return send(`https://api.spotify.com/v1/browse/categories?locale=sv_RU`, token, 'Genre');
+  }
+  /**
+   * @description Функция получения плейлистов по подходящему жанру
+   * @param {string} token - токен Spotify
+   * @param {string} genreId - id заданного жанра
+   */
+  const _getPlaylistByGenre = async (token, genreId) => {
+    return send(`https://api.spotify.com/v1/browse/categories/${genreId}/playlists?limit=10`, token, 'GenrePlaylists');
+  }
+
+  /**
+   * @description Функция получения списка песен
+   * @param {string} token - токен Spotify
+   * @param {string} tracksEndPoint - ссылка для получения API ответа с информацией про песни в плейлисте
+   */
+  const _getTracks = async (token, tracksEndPoint) => {
+
+    return send(`${tracksEndPoint}?limit=10`, token,'Songs');
+  }
+  /**
+   * @description Функция получения информации о песне
+   * @param {string} token - токен Spotify
+   * @param {string} tracksEndPoint - ссылка для получения API ответа с информацией про песню
+   */
+  const _getTrack = async (token, trackEndPoint) => {
+    return send(`${trackEndPoint}`, token);
+  }
+  /**
+   * @description Функция получения информации о плейлисте
+   * @param {string} token - токен Spotify
+   * @param {string} id - id плейлиста
+   */
+  const _getPlaylist = async (token, id) => {
+    return send(`https://api.spotify.com/v1/albums/${id}`, token);
+  }
+
+  return {
+    getToken() {
+      return _getToken();
+    },
+    getGenres(token) {
+      return _getGenres(token);
+    },
+    getPlaylistByGenre(token, genreId) {
+      return _getPlaylistByGenre(token, genreId);
+    },
+    getTracks(token, tracksEndPoint) {
+      return _getTracks(token, tracksEndPoint);
+    },
+    getTrack(token, trackEndPoint) {
+      return _getTrack(token, trackEndPoint);
+    },
+    getPlaylist(token, id) {
+      return _getPlaylist(token, id);
+    }
+  }
+})();
+
+
+const UIController = (function() {
+  const createNewElement = (type, name, img, artist) => {
+
+    const element =
+      `
+    <div class="${type}Photo">
+      <img src="${img}" class="songPhoto" alt="">
+    </div>
+    <div class="${type}Name">
+        <label for="boxes" class="SongTitle"><b>${name}</b></label>
+    </div>
+    <div class="${type}Artist">
+        <label for="boxes" class="PlaylistOwner">${artist}</label>
+    </div>
+    `
+    return element;
+  }
+
+  const DOMElements = {
+    selectGenre: '#select_genre',
+    selectPlaylist: '#select_playlist',
+    buttonSubmit: '#btn_submit',
+    divSongDetail: '#song-detail',
```

Comment:
тут можно не указывать div
```suggestion
    songDetail: '#song-detail',
```